nextcord/iterators.py

```diff
@@ -723,8 +723,12 @@ async def _retrieve_guilds_after_strategy(self, retrieve):
 
 
 class MemberIterator(_AsyncIterator["Member"]):
-    def __init__(self, guild, limit=1000, after=None):
-
+    def __init__(
+        self,
+        guild: Guild,
+        limit: Optional[int] = 1000,
+        after: Optional[Union[Snowflake, datetime.datetime]] = None,
+    ):
```

Comment:
Shouldn't #849 take care of this?

Reply:
Yes, but #849 also depends on pyright being updated, its a dual stall, so this single change can simply be merged into #849 after.